src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx

```diff
@@ -31,24 +28,35 @@ const StyledAlert = styled(Alert)`
 `;
 
 export const KnyttJournalpostTilBehandling: React.FC = () => {
+    const { toggles } = useApp();
+
     const {
         skjema,
         minimalFagsak,
-        hentSorterteBehandlinger,
+        hentSorterteJournalføringsbehandlinger,
         kanKnytteJournalpostTilBehandling,
         erLesevisning,
     } = useManuellJournalfør();
+
     const åpenBehandling: VisningBehandling | undefined = minimalFagsak
         ? hentAktivBehandlingPåMinimalFagsak(minimalFagsak)
         : undefined;
 
     const visGenerellSakInfoStripe =
         !erLesevisning() &&
-        skjema.felter.tilknyttedeBehandlingIder.verdi.length === 0 &&
+        skjema.felter.tilknyttedeBehandlinger.verdi.length === 0 &&
         !skjema.felter.knyttTilNyBehandling.verdi;
+
+    const sorterteJournalføringsbehandlinger = hentSorterteJournalføringsbehandlinger();
+
+    const skalViseKnyttTilNyBehandling =
+        toggles[ToggleNavn.kanBehandleKlage] || // Skal alltid vises når klage er implementert
```

Comment:
Litt usikker på om dette gir mening? Burde det være alltid når toggle er på OG dokumenttypen er klage?

Reply:
Enig, jeg tror vi må legge til noe ekstra kode her for å sjekke om toggelen er på OG det er klage OG behandlingen er avsluttet sant? Jeg tror også denne koden kan forbedres slik at den er enklere å lese. Kanskje legge logikken for å sjekke om en behandling er avsluttet i en egen const? Eksempelet her viser hva vi kan gjøre med mer positiv logikk, som er enklere å lese.

```
const erBehandlingAvsluttet =  !åpenBehandling || åpenBehandling.status === BehandlingStatus.AVSLUTTET;
const erKlageBehandlingAvsluttet = toggles[ToggleNavn.kanBehandleKlage] && åpenBehandling.type === Klagebehandlingstype.KLAGE && erBehandlingenAvsluttet;
const skalViseKnyttTilNyBehandling = erBehandlingAvsluttet || erKlageBehandlingAvsluttet`
```